content/src/fundings/dep-water-bank.md

```diff
@@ -37,5 +38,5 @@ descriptionMd: This program provides low-cost financing for the design,
 - **Drinking water applicants** can be public community water systems privately and publicly owned, and nonprofit non-community water systems (as defined by the [National Primary Drinking Water Regulations](https://www.epa.gov/ground-water-and-drinking-water/national-primary-drinking-water-regulations))
 - **All applicants** must also satisfy the [Water Bank's creditworthiness standards](https://cdn.njib.gov/njib/policies/njib_credit_policy_2018.pdf)
 
-> ### Benefits
+> **Benefits:**
```

Comment:
dust - I'm assuming this is an IDE change and these are equivalent

Reply:
No. Merge conflict issues. I changed it back to `### Benefits`. Thank you!